nnpdf_data/nnpdf_data/commondata/ATLAS_Z0J_8TEV/filter_utils.py

```diff
@@ -0,0 +1,284 @@
+import numpy as np
```

Comment:
Please move the utils to [nnpdf_data/nnpdf_data/filter_utils](https://github.com/NNPDF/nnpdf/tree/deda5b8c89d31eee94eeaa450d66d756a4ec0dd8/nnpdf_data/nnpdf_data/filter_utils), especially if they are re-usable.